versions/1/input/spec.yml

```
@@ -0,0 +1,42 @@
+##
+## Entrypoint of "input packages" specification.
+## The specification is considered "beta" at the moment, so it may change until we release it as GA.
+##
+## Describes the folders and files that make up a package.
+##
+spec:
+  additionalContents: true
+  totalContentsLimit: 65535
+  totalSizeLimit: 250MB
+  sizeLimit: 150MB
```

Maybe we can reduce these limits. I cannot think on a reason to have big files in inputs :thinking:
We could increase this later if we find a use case.
```suggestion
  totalSizeLimit: 50MB
  sizeLimit: 30MB
```

---

Fixed.